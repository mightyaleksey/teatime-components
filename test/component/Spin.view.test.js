'use strict';

const React = require('react');
const Spin = require('../../component/Spin');
const renderer = require('react-test-renderer');

test('defaultProp #size should have a value `m`', () => {
  expect(Spin.defaultProps.size).toBe('m');
});

test('Renders correctly', () => {
  const tree = renderer.create(
    <Spin
      className='outer'
      id='spins id'/>
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
