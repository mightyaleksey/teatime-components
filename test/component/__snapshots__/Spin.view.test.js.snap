// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<span
  className="wrapper outer"
  id="spins id"
>
  <i
    className="control"
  />
</span>
`;
